src/challenge/views.py

```diff
@@ -159,6 +209,10 @@ def post(self, request):
 
 
 class ChallengeVoteView(APIView):
+    """
+    Vote for a challenge
```

Comment:
IMO this is a bit unclear, the users aren't voting for a challenge as such, they're voting on whether they liked the challenge.